packages/rancher-gatekeeper/overlay/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+All notable changes from the upstream OPA Gatekeeper chart will be added to this file
+
+## [Package Version 00] - 2020-07-27
+### Added
+- Enabled the CRD chart generator in `package.yaml`
+
+### Modified
+- Updated chart version in `Chart.yaml` to follow the upstream's format `v3.1.0-beta.X`
```

Comment:
I feel like the changelog should only have notes on what we changed from upstream.

Reply:
Okay, I'll edit the changelog.